package-lock.json

```diff
@@ -0,0 +1,1263 @@
+{
+  "name": "st_pax_tracker",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "st_pax_tracker",
+      "dependencies": {
+        "@csstools/postcss-sass": "^5.1.1",
+        "@fortawesome/fontawesome-free": "^7.0.1",
+        "@hotwired/turbo-rails": "^8.0.16",
+        "@popperjs/core": "^2.11.8",
+        "autoprefixer": "^10.4.21",
+        "bootstrap": "^5.3.8",
+        "clipboard": "^2.0.11",
+        "datatables.net": "^2.3.4",
+        "datatables.net-bs5": "^2.3.4",
+        "datatables.net-fixedheader-bs5": "^4.0.3",
+        "jquery": "^3.7.1",
+        "postcss": "^8.5.6",
+        "postcss-cli": "^11.0.1",
+        "postcss-import": "^16.1.1",
+        "postcss-nesting": "^13.0.2",
+        "postcss-scss": "^4.0.9"
+      },
+      "devDependencies": {
+        "run-pty": "^5.0.0"
+      }
+    },
+    "node_modules/@csstools/postcss-sass": {
+      "version": "5.1.1",
+      "resolved": "https://registry.yarnpkg.com/@csstools/postcss-sass/-/postcss-sass-5.1.1.tgz",
```

Comment:
So, do we want to keep using Yarn's registry? Arguably that ensures that this change is installing _exactly_ the same packages as before, I guess.